duckduckgo/Android
# update autofill 3.0.1

Task/Issue URL: https://app.asana.com/0/414730916066338/1200553245468873/f
Tech Design URL: 
CC: 

**Description**:
Updates autofill to 3.0.1

**Steps to test this PR**:
1. Checkout branch
1. Run git submodule
1. Confirm autofill commit hash is`2d6e7a397d602653d1f1eb024d236ebb7974e30e `

**Autofill still works**
1. See https://app.asana.com/0/1198964220583541/1200553245468873

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
